lib/sparse/src/index/inverted_index/inverted_index_ram.rs

```diff
@@ -25,7 +25,24 @@ impl InvertedIndex for InvertedIndexRam {
     }
 
     fn indexed_vector_count(&self) -> usize {
-        self.postings.len()
+        // TODO(sparse) not super efficient
```

Comment:
This calculation is not efficient. But we can resolve it.
In `hnsw.rs` we can read how was implemented indexed counting. It counts existing points only in `build` method and does not change the saved value. We can do the same, in `SparseVectorIndex` precalculate indexed count in `build` method and reuse it